lighthouse-core/computed/load-simulator.js

```diff
@@ -29,10 +29,26 @@ class LoadSimulator {
     // If we have precomputed lantern data, overwrite our observed estimates and use precomputed instead
     // for increased stability.
     if (precomputedLanternData) {
-      options.additionalRttByOrigin = new Map(Object.entries(
-        precomputedLanternData.additionalRttByOrigin));
-      options.serverResponseTimeByOrigin = new Map(Object.entries(
-        precomputedLanternData.serverResponseTimeByOrigin));
+      const additionalRttTimes = Object.entries(precomputedLanternData.additionalRttByOrigin);
+      const serverResponseTimes = Object.entries(precomputedLanternData.serverResponseTimeByOrigin);
+
+      options.additionalRttByOrigin = new Map(additionalRttTimes);
+      options.serverResponseTimeByOrigin = new Map(serverResponseTimes);
+
+      // apply wildcard entry to all origins
+      if ('*' in precomputedLanternData.additionalRttByOrigin) {
```

Comment:
could we reverse the order so the wildcard is the base and it can be overriden by individual origin latencies?